db/migrate/20191018164845_add_column_returned_at_to_rents.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnReturnedAtToRents < ActiveRecord::Migration[5.1]
+  def change
+    add_column :rents, :returned_at, :date
```

Comment:
Where was this required? I think you can rollback this  migration

Reply:
Done